dockerfiles/force-readthedocs-addons.js

```diff
@@ -0,0 +1,219 @@
+/*
+
+  Script to inject the new Addons implementation on pages served by El Proxito.
+
+  This script is ran on a Cloudflare Worker and modifies the HTML with two different purposes:
+
+      1. remove the old implementation of our flyout (``readthedocs-doc-embed.js`` and others)
+      2. inject the new addons implementation (``readthedocs-addons.js``) script
+
+  Currently, we are doing 1) only when users opt-in into the new beta addons.
+  In the future, when our addons become stable, we will always remove the old implementation,
+  making all the projects to use the addons by default.
+
+*/
+
+// add "readthedocs-addons.js" inside the "<head>"
+const addonsJs =
+  '<script async type="text/javascript" src="/_/static/javascript/readthedocs-addons.js"></script>';
```

Comment:
If we want to test an addon we are developing, we can just change this URL to `http://localhost:8000/readthedocs-addons.js` while having `npm run dev` running and refresh the page (there is no need to restart Docker or anything) -- 💯

Reply:
Note, that there is a `inv docker.up -w` flag that enables webpack hosting. This would be a good pattern to follow eventually. Making this a manual operation means it won't be repeatable or easy to use.

The development workflow with Webpack dev server hosting these assets is really good and should probably be the default for core team.

Reply:
Yes. I'm happy to create an issue to track this work. I don't think we have a pattern for "given an environment variable make a decision on an asset file". We will need to change the content of this file in particular, so, it may require some research.

Reply:
https://github.com/readthedocs/common/issues/199